go/src/koding/api/apitest/apitest.go

```diff
@@ -31,20 +36,33 @@ type HTTPIdleConnectionsCloser interface {
 	CloseIdleConnections()
 }
 
+// FakeAuth is a mock which provides fake authentication server.
+//
+// It implements both client-facing interface  the Auth method -
+// than can be used with clients' transports, and also a server-facing
+// one - the GetSession method - which is a handler that can be
+// used with http server.
 type FakeAuth struct {
 	Sessions map[string]*api.Session
 
 	mu sync.RWMutex
 }
 
-var _ http.Handler = http.HandlerFunc((&FakeAuth{}).GetSession)
+var (
+	_ http.Handler = http.HandlerFunc((&FakeAuth{}).GetSession)
```

Comment:
The same applies to any other type - you could argue that blank interface variables are not needed, because a test or package elsewhere won't build.

Those blanks are also for documentation purpose, so you see in one variable group what kind of functionality does the type serve.